server/src/routes/user/delete.ts

```diff
@@ -4,24 +4,26 @@ import { deleteConnectedUserByUID } from "../../actions/deleteConnectedUser";
 const deleteUserRoute = Router();
 
 deleteUserRoute.post("/users", async (req, res) => {
```

Comment:
This actually needs to be a DELETE method not a POST method??
Also a pre-existing issue but I would like this to be fixed now

Reply:
Whoops I tried to make the change here but I accidentally put it in another PR which I plan to merge after this one is merged. Anyways, this is now fixed.
https://github.com/ufosc/OSC-Proximity-Chat-App/pull/273/commits/38312a418740299b11b7568b2a303f8db33a548a